src/server/routes/entity/entity.tsx

```diff
@@ -84,67 +84,68 @@ export function displayEntity(req: PassportRequest, res: $Response) {
 
 	let editorEntityVisitPromise;
```

Comment:
We need to rename all these xyzPromise variables; since we are not assigning a promise to them but instead `await`ing a promise and assigning the returned value directly, they should be
`editorEntityVisitPromise` -> `editorEntityVisit`
`alertPromise` -> `alert` 
etc.